assignments/this.js

```diff
@@ -1,10 +1,10 @@
 /* The for principles of "this";
  * in your own words. explain the four principle for the "this" keyword below.
  *
- * 1.
- * 2.
- * 3.
- * 4.
+ * 1. Implicit this used in methods
```

Comment:
If the object contains a reference to this function in its scope, and when calling the function 
it is preceded by the object reference (i.e.: obj.function()), than this object should be used for the functions 'this' binding (i.e.: this = obj).
The object in implicit binding have to contain a reference in itself to the function!